src/openzaak/import_data/admin.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2024 Dimpact
+from django.contrib import admin
+
+from openzaak.import_data.models import Import
+
+
+@admin.register(Import)
+class ImportAdmin(admin.ModelAdmin):
```

Comment:
please add `PrivateMediaMixin` so urls to download files and reports will work. `EnkelvoudigInformatieObjectAdmin` can be served as inspiration